assets/app/views/directives/logs/_log-formatted.html

```diff
@@ -10,7 +10,7 @@
       affix
       offset-top="285"
       offset-bottom="90">
-      <a ng-click="scrollBottom()">
+      <a ng-href="#" ng-click="scrollBottom()">
```

Comment:
We should probably make these `href=""`

https://docs.angularjs.org/api/ng/directive/a